packages/api/tests/endpoints/test-granules.js

```diff
@@ -107,7 +107,7 @@ process.env.ExecutionsTable = randomId('executions');
 process.env.CollectionsTable = randomId('collection');
 process.env.GranulesTable = randomId('granules');
 process.env.stackName = randomId('stackname');
-process.env.system_bucket = randomId('system-bucket');
+process.env.system_bucket = randomId('bucket');
```

Comment:
I believe when I made the change I was a wholesale find/replace on what I thought were invalid bucket names, because the newer version of Localstack requires only valid bucket names. But this one probably isn't invalid? I can revert

Reply:
I don't think it needs reverting, that was mostly curiosity on my part.